translations/ui-orders/en.json

```diff
@@ -741,6 +742,7 @@
   "settings.newBtn": "New",
   "settings.routing.label": "Routing",
   "settings.routing.address": "Routing address",
+  "settings.routing.address.errors.permissionsRequired": "User does not have required permissions.",
```

Comment:
if you totally removed error handling, this and error above are not used

Reply:
removed both message ids